lib/assessment.js

```diff
@@ -4,23 +4,33 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(nums) {
+  return Math.min(nums)
```

Comment:
This will not work, because `nums` now refers to only the _first_ argument passed in. 

(it also kinda misses the point, which is to see how you might implement `Math.min` on your own)